### PR TITLE
 Translate tutorials/clusters/apparmor into Japanese

### LINKED ISSUE #26953: Translate tutorials/clusters/apparmor into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate [tutorials/clusters/apparmor](https://kubernetes.io/docs/tutorials/clusters/apparmor) into Japanese.

**Why is this needed**

This page has not been translated.

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by shuuji3**
Translate [tutorials/clusters/apparmor](https://kubernetes.io/docs/tutorials/clusters/apparmor) into Japanese.

**Page preview**: https://deploy-preview-27384--kubernetes-io-master-staging.netlify.app/ja/docs/tutorials/clusters/apparmor/

Closes #26953.

### CODE REVIEW

File: content/ja/docs/tutorials/clusters/apparmor.md [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+```
+
+AppArmorのオプションはPodSecurityPolicy上でアノテーションとして指定します。
+
+```yaml
+apparmor.security.beta.kubernetes.io/defaultProfileName: <profile_ref>
+apparmor.security.beta.kubernetes.io/allowedProfileNames: <profile_ref>[,others...]
+```
+
+defaultProfileNameオプションには、noneが指定された場合にコンテナにデフォルトで適用されるプロファイルを指定します。allowedProfileNamesオプションには、Podコンテナの実行が許可されるプロファイルのリストを指定します。両方のオプションが指定された場合、デフォルトは許可されなければいけません。プロファイルはコンテナ上で同じフォーマットで指定されます。完全な仕様については、[APIリファレンス](#api-reference)を参照してください。
```
Comment (atoato88) [inline content/ja/docs/tutorials/clusters/apparmor.md:305]: This is just the question about meaning of `none` at here in Japanese.
Is it rather `何も指定されない` than `noneが指定された` ?

File: content/ja/docs/tutorials/clusters/apparmor.md [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+  - Dockerの場合、非特権コンテナでは[`docker-default`](https://docs.docker.com/engine/security/apparmor/)プロファイルが選択され、特権コンテナではunconfined(プロファイルなし)が選択されます。
+- `localhost/<profile_name>`: 名前で指定されたノード(localhost)に読み込まれたプロファイルを指します。
+  - 利用できるプロファイル名の詳細は[core policy reference](https://gitlab.com/apparmor/apparmor/wikis/AppArmor_Core_Policy_Reference#profile-names-and-attachment-specifications)で説明されています。
+- `unconfined`: これは実質的にコンテナ上のAppArmorを無効化します。
+
+これ以外のプロファイルリファレンスはすべて無効です。
+
+### PodSecurityPolicyアノテーション
+
+noneが与えられたときにコンテナに適用するデフォルトのプロファイルは、以下のように指定します。
```
Comment (atoato88) [inline content/ja/docs/tutorials/clusters/apparmor.md:359]: Same comment with [here](https://github.com/kubernetes/website/pull/27384/files#r607549412).

File: content/ja/docs/tutorials/clusters/apparmor.md (hunk 1 above)
Comment (shuuji3) [reply content/ja/docs/tutorials/clusters/apparmor.md]: Thanks for your review. It's my bad. I think your suggestion is right.😄  Fixed it.

File: content/ja/docs/tutorials/clusters/apparmor.md (hunk 2 above)
Comment (shuuji3) [reply content/ja/docs/tutorials/clusters/apparmor.md]: I fixed this too.

File: content/ja/docs/tutorials/clusters/apparmor.md (hunk 1 above)
Comment (atoato88) [reply content/ja/docs/tutorials/clusters/apparmor.md]: Thank you to update!
Perfect!! :smile:

File: content/ja/docs/tutorials/clusters/apparmor.md (hunk 2 above)
Comment (atoato88) [reply content/ja/docs/tutorials/clusters/apparmor.md]: Thank you. I confirmed that.

**Comment by atoato88** [comment]
/lgtm
It's OK for me and thank you to quick update!

**Pull request merged**